scene/gui/text_edit.cpp

```diff
@@ -463,17 +463,22 @@ void TextEdit::_notification(int p_what) {
 
 		case NOTIFICATION_INTERNAL_PHYSICS_PROCESS: {
 			if (scrolling && get_v_scroll() != target_v_scroll) {
-				double target_y = target_v_scroll - get_v_scroll();
-				double dist = abs(target_y);
-				// To ensure minimap is responsive override the speed setting.
-				double vel = ((target_y / dist) * ((minimap_clicked) ? 3000 : v_scroll_speed)) * get_physics_process_delta_time();
+				double delta_y = target_v_scroll - get_v_scroll();
+				double dist = abs(delta_y);
+				// To ensure minimap is responsive, override the speed setting.
+				double vel = SIGN(delta_y) * (minimap_clicked ? 3000 : v_scroll_speed) * get_physics_process_delta_time();
```

Comment:
The 3000 value should likely be computed relatively to the `v_scroll->get_step()` value, or relatively to the height of the scrollable area.

Reply:
Tried this and it didn't look nice IMO. It should look near-instant in any case, and I found it awkward when the line you clicked was nearby.